bindings/nodejs/tests/suites/index.mjs

```diff
@@ -36,7 +36,11 @@ export function runner(testName, scheme) {
     config.root = generateRandomRoot(config.root)
   }
 
-  const operator = scheme ? new Operator(scheme, config) : null
+  let operator = scheme ? new Operator(scheme, config) : null
+
+  let retryLayer = new RetryLayer(true, 4)
```

Comment:
The API doesn't look good to me.

Reply:
Do you have any suggestions?
Like providing params as a dic like `{jitter:ture, maxTimes: 4}`?